python/frontend/cirrus/cirrus/core.py

```diff
@@ -207,7 +216,7 @@ def launch_error_task(self, command_dict=None):
 
     def launch_ps(self, command_dict=None):
         cmd = 'nohup ./parameter_server --config config_%d.txt --nworkers %d --rank 1 --ps_port %d &> ps_out_%d & ' % (
-            self.ps_ip_port, self.n_workers * 2, self.ps_ip_port, self.ps_ip_port)
+            self.ps_ip_port, self.n_workers * 100, self.ps_ip_port, self.ps_ip_port)
```

Comment:
Why is this ```* 100```?

Reply:
Changing it back to 2. There is no reason that it should be 100.